### PR TITLE
remove-limiter

### PR DESCRIPTION
**Pull request by TakenPilot**
Patch
- Remove the limiter on list entries

### CODE REVIEW

**Comment by yoshokatana** [comment]
limiter?

**Comment by TakenPilot** [comment]
limiter.

File: lib/services/db.js [hunk 1]
```diff
@@ -89,12 +89,12 @@ function del(key) {
...
  * `/components/text-box`, so the search should really be `/components/text/`.
  *
- * @param {object} [options]  Defaults to limit of 10.
+ * @param {object} [options]
  * @returns {ReadStream}
  */
 function list(options) {
   options = _.defaults(options || {}, {
-    limit: 10,
+    limit: -1,
```
Comment (TakenPilot) [inline lib/services/db.js]: limiter...

**Comment by yoshokatana** [comment]
:+1: NO LIMIT

**Comment by TakenPilot** [comment]
_no limit_

**Comment by mrandhahn** [comment]
"The only limit is yourself" - zombo.com

**Comment by yoshokatana** [comment]
@mrandhahn :shipit:

**Pull request merged**